### PR TITLE
Add competition screen, styling and formating data.

### PR DESCRIPTION
**Pull request by qingxin-yuan**
Add competition screen, with styling and data formatting. Navigation is also added to the competition stack.

New icon assets added in competition and competition details folder.

### CODE REVIEW

File: js/screens/CompetitionCategory/CompetitionCategory.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+/* global require */
```
Comment (siddharthparmar7) [inline js/screens/CompetitionCategory/CompetitionCategory.js:1]: Please remove this comment.

File: js/screens/CompetitionList/CompetitionList.js [hunk 2]
```diff
@@ -33,8 +43,9 @@ export default CompetitionList;
 
 CompetitionList.propTypes = {
   list: PropTypes.array.isRequired,
+  navigation: PropTypes.object.isRequired
 };
 
 CompetitionList.defaultProps = {
-  list: [],
-}
+  list: []
```
Comment (siddharthparmar7) [inline js/screens/CompetitionList/CompetitionList.js]: if your propType is required, you don't need default prop for that. If you have questions, please ask.

File: js/screens/CompetitionList/CompetitionListContainer.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 import { firebaseDB } from "../../config/firebaseConfig";
 import CompetitionList from "./CompetitionList";
 import { competitionValidation } from "../../helpers/timestampHelpers";
+import { headerBarStyle } from "../../config/styles";
 
 class CompetitionListContainer extends Component {
+  static navigationOptions = ({ navigation }) => ({
+    title: navigation.state.params.category.toUpperCase() || "COMPETITION",
+    ...headerBarStyle(navigation)
+    // headerLeft: null
```
Comment (siddharthparmar7) [inline js/screens/CompetitionList/CompetitionListContainer.js]: Please remove this comment.

File: js/screens/CompetitionList/CompetitionListContainer.js [hunk 4]
```diff
@@ -34,15 +40,25 @@ class CompetitionListContainer extends Component {
 
   componentDidMount() {
     // TODO: Change this param to dynamic, once navigation is set up
-    const param = "";
+    // const param = "";
```
Comment (siddharthparmar7) [inline js/screens/CompetitionList/CompetitionListContainer.js]: Please remove this comment.

File: js/screens/CompetitionList/CompetitionList.js [hunk 5]
```diff
@@ -1,28 +1,38 @@
...
+import { timestampConvertion } from "../../helpers/timestampHelpers";
 
 import { styles } from "./styles";
 
-const CompetitionList = ({ list }) => {
+const CompetitionList = ({ list, navigation }) => {
   return (
     <View style={styles.main}>
-      {list.map((el, i) => {
+      {list.map((comp, i) => {
```
Comment (siddharthparmar7) [inline js/screens/CompetitionList/CompetitionList.js]: instead of comp use competition. It is better naming.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes so I can merge your branch.

**Comment by qingxin-yuan** [comment]
Thanks for the feedback! I updated the code. 
PS: The line `//global require` is for getting rid of eslint errors.

Cheers

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**